frontend/apps/filemanager/filemanager.lua

```diff
@@ -88,6 +92,11 @@ function FileManager:init()
     }
     self.file_chooser = file_chooser
 
+    function file_chooser:onPathChanged(path)  -- luacheck: ignore
+        FileManager.instance.banner[2].text = path
```

Comment:
Sorry this is not documented, but it's better to use ` FileManager.instance.banner[2]:setText(path)` here.